pm.mirko.Atoms.yml

```diff
@@ -0,0 +1,236 @@
+---
+app-id: pm.mirko.Atoms
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: atoms
+
+finish-args:
+- "--share=network"
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--device=dri"
+- "--socket=wayland"
+- "--allow=devel"  # this make proot works
+# - "--talk-name=org.freedesktop.Flatpak"  # set this for distrobox support
+
+add-extensions:
+  org.gnome.Platform.Compat.i386:
```

Comment:
why do you need this?

Reply:
Isn't this for 32-bit binaries compatibility? Those are chroot(s) which are supposed to run both 64/32 bit apps.

Reply:
I'm asking you, you are the one submitting. 

But my understanding is that the "chroot" would bring it's own runtime and doesnt need this.

Reply:
It didn't work previously without this extension but I just tried again and now it works, I must have unintentionally fixed with some update.